composer/datasets/streaming/format.py

```diff
@@ -21,25 +23,57 @@
 ]
 
 
-def get_index_basename() -> str:
+def split_compression_suffix(local_path: str) -> Tuple[str, Optional[str]]:
```

Comment:
Can this be a private helper?

Reply:
I don't believe so - it's used in several different files